AdminApi/src/AdminApi.Infrastructure.Database.Postgres/Migrations/20240315123223_Update_TiersOverview_NewProperties.cs

```diff
@@ -0,0 +1,34 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+#nullable disable
+
+namespace Backbone.AdminApi.Infrastructure.Database.Postgres.Migrations
+{
+    /// <inheritdoc />
+    public partial class Update_TiersOverview_NewProperties : Migration
+    {
+        /// <inheritdoc />
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql("""
+                CREATE OR REPLACE VIEW "TierOverviews" AS
+                    SELECT
+                        TIERS."Id" AS "Id",
+                        TIERS."Name" AS "Name",
+                        COUNT (IDENTITIES."TierId") AS "NumberOfIdentities",
+                        TIERS."CanBeUsedAsDefaultForUser" AS "CanBeUsedAsDefaultForUser",
+                        TIERS."CanBeManuallyAssigned" AS "CanBeManuallyAssigned"
+                    FROM "Devices"."Tiers" TIERS
+                    LEFT JOIN "Devices"."Identities" IDENTITIES
+                    ON IDENTITIES."TierId" = TIERS."Id"
+                    GROUP BY TIERS."Id", TIERS."Name", TIERS."CanBeUsedAsDefaultForUser", TIERS."CanBeManuallyAssigned"
```

Comment:
I'm uncertain if this `GROUP BY` statement is correct. @daniel-almeida-konkconsulting, could you have a look please?

Reply:
I made a mistake in my thinking. Ignore this comment.